client/src/rpc_client.rs

```diff
@@ -1247,6 +1248,34 @@ impl RpcClient {
         )
     }
 
+    pub fn get_fees(&self) -> ClientResult<Fees> {
```

Comment:
I'm hoping there might be an opportunity to make these methods consistent with those in BanksClient at some point...